README.md

```diff
@@ -147,6 +147,22 @@ mosh-client packages to allow mosh-server usage without Perl.
 Notes for developers
 --------------------
 
+To start contributing to Mosh, install the following dependencies:
```

Comment:
Can you mention that this is for a Debian or Ubuntu machine, possibly on  WSL?

Reply:
Done